src/libraries/System.Security.Cryptography.Algorithms/tests/DefaultECDiffieHellmanProvider.Unix.cs

```diff
@@ -34,7 +34,7 @@ public bool ExplicitCurvesSupported
             }
         }
 
-        public bool CanDeriveNewPublicKey => true;
+        public bool CanDeriveNewPublicKey => !PlatformDetection.IsiOS && !PlatformDetection.IstvOS;
```

Comment:
No reason to recompute it over and over.

```suggestion
        public bool CanDeriveNewPublicKey { get; } = !PlatformDetection.IsiOS && !PlatformDetection.IstvOS;
```